yarn.lock

```diff
@@ -2491,6 +2491,11 @@
     "@typescript-eslint/eslint-plugin" "^4.14.0"
     "@typescript-eslint/parser" "^4.14.0"
 
+"@standardnotes/features@1.0.0":
```

Comment:
I'm guessing we'll update the version when the dependents are updated?

Reply:
this is for `@standardnotes/snjs` package ... no need to update the features dependcy version in there yet, but we can do it soon